src/styles/index.scss

```diff
@@ -2,2 +1,247 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white-accent: #fff;
+$card-title-color: #34568b;
+
+.header {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-accent;
   margin: 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+  box-sizing: border-box;
+
+  &__logo {
+    width: 40px;
+    height: 40px;
+    display: flex;
+    align-items: center;
+    margin-top: 2px;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__links {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  &__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    line-height: 60px;
+  }
+
+  &__item {
+    display: inline-block;
+    margin-left: 17px;
+    text-align: center;
+    height: 60px;
+    line-height: 60px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    cursor: pointer;
+    display: flex;
+    color: black;
```

Comment:
The color 'black' is hardcoded here. Consider using a variable for consistency and maintainability of your color scheme.